prog/postgres/converge_postgres_resource.rb

```diff
@@ -14,8 +14,11 @@ class Prog::Postgres::ConvergePostgresResource < Prog::Base
     hop_wait_servers_to_be_ready if postgres_resource.has_enough_fresh_servers?
 
     if postgres_resource.servers.all? { _1.vm.vm_host } || postgres_resource.location.provider == "aws"
-      # vm_host might be nil for AWS VM based PG instances
-      exclude_host_ids = (Config.development? || Config.is_e2e) ? [] : postgres_resource.servers.map { _1.vm.vm_host&.id }.compact
+      exclude_host_ids = []
+      if !(Config.development? || Config.is_e2e) && postgres_resource.location.provider == HostProvider::HETZNER_PROVIDER_NAME
+        used_data_centers = postgres_resource.servers.map { _1.vm.vm_host.data_center }.uniq
```

Comment:
jw: does the allocator do some balancing among different data centres? Or do we have enough spare capacity in 3 different DCs always to not worry about this?

Reply:
Allocator does not do any balancing, but I think we are big enough (in Hetzner) to not worry about this. This returns many hosts from many different data centers. (I'm not sharing the output as it is not a public information)

` VmHost.all.map{ _1.data_center }.tally`